code/datums/spells/summonitem.dm

```diff
@@ -112,11 +112,16 @@
 					if(!target.equip_to_slot_if_possible(item_to_retrieve, slot_l_hand, FALSE, TRUE))
 						butterfingers = 1
 			if(butterfingers)
-				item_to_retrieve.loc = target.loc
-				item_to_retrieve.loc.visible_message("<span class='caution'>\The [item_to_retrieve] suddenly appears!</span>")
-				playsound(get_turf(target),'sound/magic/summonitems_generic.ogg',50,1)
+				if(isturf(target.loc))
+					item_to_retrieve.loc = target.loc
+					item_to_retrieve.loc.visible_message("<span class='caution'>[item_to_retrieve] suddenly appears!</span>")
+					playsound(get_turf(target),'sound/magic/summonitems_generic.ogg',50,1)
+				else
+					item_to_retrieve.loc.visible_message("<span class='caution'>[item_to_retrieve] fails to appear!</span>")
+					playsound(get_turf(target),'sound/magic/summonitems_generic.ogg',50,1)
+					item_to_retrieve.loc.visible_message("<span class='warning'>[item_to_retrieve] reappears in an instant!</span>")
```

Comment:
Looking more at the logic, the "is it possible" check should really happen up above and then early return if it fails, otherwise the item never goes anywhere, but you instantly get 2 messages about it.

Reply:
Fair, I'll rewrite this in a sec